client/src/Components/Pages/Search/index.js

```diff
@@ -0,0 +1,220 @@
+import React, { Component } from "react";
+import axios from "axios";
+import Autosuggest from "react-autosuggest";
+
+// UI helper functions
+import {
+  SVGCreator,
+  StarRateCreator,
+  SortArrayNewest
+} from "../../../UIHelper";
+
+// styles
+import {
+  HeadlineDiv,
+  RowDiv,
+  ItemDiv,
+  SearchLegendDiv,
+  SymbolDiv,
+  OrganisationDetailsDiv,
+  ReviewDetailsDiv,
+  InnerDivLastReviews,
+  InnerDivSuggestions,
+  ArrowDiv,
+  SearchWrapper,
+  SuggestionBox,
+  AddItemBox,
+  LegendTitle,
+  ReviewsFrame,
+  ProfileLink
+} from "./Search.style";
+
+import { organizationIcons } from "./../../../theme";
+
+export default class Search extends Component {
+  state = {
+    loaded: false,
+    data: null,
+    value: "",
+    suggestions: []
+  };
+
+  componentDidMount() {
+    axios.get("/api/search").then(organizations => {
```

Comment:
Lets create a file called `apiUrls.js` which is shared between the client and the server.

```
const API_SEARCH_URL = '/api/search'

module.exports = {
	API_SEARCH_URL
}
```